_data/releases.tsv

```diff
@@ -37,3 +37,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2020-04-27	prod81	0.8.1	12579	1195994	1163031	9084406	63757664	169.25	23.18	366
 2020-05-19	prod82	0.8.2	12617	1210586	1177623	9098998	63786848	169.48	23.22	367
 2020-06-15	prod83	0.8.3	12617	1210586	1177623	9098998	63786848	169.48	23.22	367
+2020-06-30	prod84	0.8.4	12935	1210586	1178081	9104040	64886054	180.8749	23.23	
```

Comment:
Database size is `358G` (probably some shrinkage due to the PSGL 11 dump/restore process ?)